mrt/meetings/printouts.py

```diff
@@ -37,7 +38,23 @@ class ProcessingFileList(PermissionRequiredMixin, MethodView):
 
     def get(self):
         page = request.args.get('page', 1, type=int)
-        jobs = Job.query.filter_by(meeting=g.meeting).order_by(desc(Job.date))
+        user = User.query.get(current_user.get_id())
```

Comment:
This is not necessary. `current_user` has the same properties as a `User`. 
https://flask-login.readthedocs.io/en/latest/#flask.ext.login.current_user